### DISCUSSION TITLE
Invalid HTTP request received when making request from NextJS front-end server proxy

### DISCUSSION THREAD

**Discussion by lukemasonIC** [fastapi/fastapi#11490] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi.middleware.cors import CORSMiddleware

app = FastAPI()

origins = ["*"]

app.add_middleware(
    CORSMiddleware,
    allow_origins=origins,
    allow_credentials=True,
    allow_methods=["*"],
    allow_headers=["*"],
)
```

### Description

I'm running a NextJS server proxy in my front-end application to make requests to a FastAPI back-end. I have set the CORS wildcard and read through the docs but still receive the invalid http request error. I imagine it has to do with CORS, but I still can't seem to get it working. Any help would be great, thank you!

### Operating System

macOS

### FastAPI Version

0.100.0

### Pydantic Version

2.0.3

### Python Version

Python 3.11.7

**Comment by YuriiMotov** [comment]
Have you read about the limitation of wildcards in CORS origins? https://fastapi.tiangolo.com/tutorial/cors/#wildcards

> But that will only allow certain types of communication, excluding everything that involves credentials: Cookies, Authorization headers like those used with Bearer Tokens, etc.

Have you tried to specify front-end origin explicitly?

**Reply by lukemasonIC** [reply]
Hi @YuriiMotov! I have tried specific origins as well and am not seeing any luck with that either

**Reply by YuriiMotov** [reply]
And what about the error? Can you see the status code or any other information about the error?
What do you see on back-end server console\log?

**Comment by YuriiMotov** [comment]
Closing this. Feel free to reopen it when you are ready to provide more info
